src/wheel/_bdist_wheel.py

```diff
@@ -218,7 +220,8 @@ def finalize_options(self) -> None:
         self.set_undefined_options("bdist", *zip(need_options, need_options))
 
         self.root_is_pure = not (
-            self.distribution.has_ext_modules() or self.distribution.has_c_libraries()
+            self.distribution.has_ext_modules()  # type: ignore[attr-defined]
```

Comment:
Couldn't we just add a variable annotation for `distribution` in the class?

Reply:
I thought of doing that, but the correct type to use for it isn't annotated so it causes more type problems on `distribution`'s attributes.  I could add `distribution: Any` to the class but then Mypy won't say when that isn't needed anymore.  I settled on using `# type: ignore` for these.